posts/2024-06-07_admiral_1.1_is.../admiral_1.1_is_here!.qmd

```diff
@@ -0,0 +1,145 @@
+---
+title: "{admiral} 1.1 is here!"
+author:
+  - name: Edoardo Mancini
+  - name: Ben Straub
+description: "Get to know what awaits you with the new release of the {admiral} package, including enhanced error messaging, improved documentation and much more!"
+date: "2024-06-07"
+# please do not use any non-default categories.
+# You can find the default categories in the repository README.md
+categories: [ADaMs]
+# feel free to change the image
+image: "admiral.png"
+
+---
+
+<!--------------- typical setup ----------------->
+
+```{r setup, include=FALSE}
+long_slug <- "2024-06-07_{admiral}_1.1_is..."
+# renv::use(lockfile = "renv.lock")
+```
+
+<!--------------- post begins here ----------------->
+
+[`admiral 1.1.0`](https://pharmaverse.github.io/admiral/index.html) is out on [CRAN](https://cran.r-project.org/web/packages/admiral/index.html)! 
+Though it may seem like just yesterday that `{admiral}` achieved its [milestone 1.0 release](https://pharmaverse.github.io/blog/posts/2023-12-18_admiral_1_0/admiral_1_0.html), in actual fact six long months have gone by - testament to our commitment to now release twice-yearly rather than quarterly. During this time, we in the `{admiral}` team received lots of positive feedback from our community, but have nevertheless worked tirelessly to improve our package. This new release comes with various quality of life changes, targeted additions to our functions' capabilities and  improvements to our documentation that all contribute to improve each and every user's experience - all while ensuring that our commitment to stability and a low amount of breaking changes is still met.
+
+Thi blog post will showcase some of the highlights of this release, but you can explore the full contents in our [Changelog](https://pharmaverse.github.io/admiral/news/index.html).
+
+# Improved error messaging
+
+Our marquee update for `{admiral}` 1.1 is certainly our new and improved error messaging. Through a deep rework of our `{admiraldev}` package we have completely reworked how errors and warnings are thrown by `{admiral}` - all without creating any breaking changes for our users. In our new error messages:
+
+* The name of the calling function appears on the top line to orient the user where the error is coming from.
+* Color coding of words is now present so that the messages are easier on the eyes and easier to read.
+* When applicable, the message is followed by an informative message (with the standard `i` prefix) providing details on what to do to address the error.
```

Comment:
```suggestion
* When applicable, the error is followed by an informative message (with the standard `i` prefix) providing details on what to do to address the error.
```

Reply:
should we not be more general, so that we can include warnings as well?